src/test/resources/com/google/jenkins/plugins/computeengine/configuration-as-code.yml

```diff
@@ -32,7 +32,9 @@ jenkins:
                 region:         europe-west1
                 subnetworkShortName: gce-jenkins-cloud
             networkTags:        jenkins-agent
-            externalAddress:    true
+            networkInterfaceIpStackMode:
+              singleStack:
+                externalIPV4Address:    true
```

Comment:
Note: incompatible for existing setups.

Reply:
Yes, IIRC we merged this fix: https://github.com/jenkinsci/google-compute-engine-plugin/pull/440, to make it compatible.

Reply:
No, I am referring to the case that a cloud is defined using CasC and did not mention `externalAddress` (so using the default of `false`). After the update, the bundle still loads, but the cloud throws NPEs when it tries to provision VMs because the `networkInterfaceIpStackMode` field is unset. (In general, the CasC support in this plugin is very poor—tons of fields lack sensible defaults, so you need a very verbose document to get anything to work, and the documentation is inadequate so you need to use trial and error.)

Reply:
:thinking: 
ok! if you want I can try to look at it